modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/tests/WorkflowMetricsAllItemsList.testcase

```diff
@@ -2982,9 +2982,123 @@ definition {
 	@description = "LPS-98517 - Verify that the usernames in the result table are separated by a comma when there is more than one assignee per item"
 	@priority = 3
 	test UsernamesAreSeparatedByComma {
+		property portal.acceptance = "true";
+
+		task ("Given a primary user who creates a secondary Power user with a membership") {
+			WorkflowMetrics.executeIndexActions(actionDescription = "Workflow Indexes");
+
+			JSONUser.addUser(
+				userEmailAddress = "tester@liferay.com",
+				userFirstName = "tester",
+				userLastName = "tester",
+				userScreenName = "tester");
+
+			JSONUser.setFirstPassword(
+				agreeToTermsAndAnswerReminderQuery = "true",
+				requireReset = "false",
+				userEmailAddress = "tester@liferay.com");
+
+			JSONRole.assignRoleToUser(
+				roleTitle = "Administrator",
+				userEmailAddress = "tester@liferay.com");
+
+			JSONRole.assignRoleToUser(
+				roleTitle = "Power User",
+				userEmailAddress = "tester@liferay.com");
+
+			Navigator.openURL();
+
+			ApplicationsMenu.gotoPortlet(
+				category = "Configuration",
+				panel = "Control Panel",
+				portlet = "Instance Settings");
+
+			PortalSettings.gotoConfiguration(
+				configurationCategory = "User Authentication",
+				configurationName = "General",
+				configurationScope = "Virtual Instance Scope");
+
+			PortalSettings.uncheckStrangersVerifyEmailCheckbox();
```

Comment:
@kristinonias hey Kristin, your test shouldn't need this because CI sets this to false by default